src/main/java/co/rsk/federate/signing/hsm/message/PowHSMSignerMessage.java

```diff
@@ -18,33 +18,71 @@
 
 package co.rsk.federate.signing.hsm.message;
 
+import static co.rsk.federate.signing.HSMField.INPUT;
+import static co.rsk.federate.signing.HSMField.OUTPOINT_VALUE;
+import static co.rsk.federate.signing.HSMField.SIGHASH_COMPUTATION_MODE;
+import static co.rsk.federate.signing.HSMField.TX;
+import static co.rsk.federate.signing.HSMField.WITNESS_SCRIPT;
+
 import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.bitcoinj.core.Coin;
 import co.rsk.bitcoinj.core.Sha256Hash;
+import co.rsk.bitcoinj.core.TransactionWitness;
+import co.rsk.bitcoinj.core.Utils;
 import co.rsk.trie.Trie;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
+import java.util.Optional;
 import org.ethereum.core.TransactionReceipt;
 import org.spongycastle.util.encoders.Hex;
 
 public class PowHSMSignerMessage extends SignerMessage {
+
+    public static final int OUTPOINT_VALUE_LE_FORMAT_SIZE_IN_BYTES = 8;
+    public static final int HSM_VERSION_5 = 5;
+    public static final String SIGHASH_LEGACY_MODE = "legacy";
+    public static final String SIGHASH_SEGWIT_MODE = "segwit";
```

Comment:
why public?

Reply:
Just to be able to use it in the test assertions. But it's ok, I agree that better to duplicate them there.

Reply:
Updated.